app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h3><%= @recipe.name %></h3>
+
+<%= "Complexity: #{@recipe.complexity}" %>
+<%= "Genre: #{@recipe.genre}" %>
+<%= "Ingredients: #{@recipe.all_ingredients}" %>
+<%= "Total Cost: #{@recipe.total_cost}" %>
+
+<%= form_with url: "/recipes/#{@recipe.id}/recipe_ingredients", method: :post, local: true do |form| %>
```

Comment:
Super RESTful route, nice work!

It doesn't matter either way, but you could make the path `/recipes/:recipe_id/ingredients` too and it still communicates that it's ingredients for a recipe. But yours is totally fine